autogpt_platform/frontend/src/app/(platform)/copilot/components/PulseChips/PulseChips.module.css

```
@@ -0,0 +1,72 @@
+  animation: rotateBorder 6s linear infinite;
+  pointer-events: none;
+  z-index: -1;
+}
+
+@property --border-angle {
+  syntax: "<angle>";
+  initial-value: 0deg;
+  inherits: false;
+}
+
+@keyframes rotateBorder {
```

Stylelint (17.6.0)

[error] Expected keyframe name "rotateBorder" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)